cmd/mistry/main.go

```diff
@@ -187,40 +212,24 @@ EXAMPLES:
 					}
 				}
 
-				jr := types.JobRequest{Project: project, Group: group, Params: params}
-				jrJSON, err := json.Marshal(jr)
-				if err != nil {
-					return err
+				url := fmt.Sprintf("http://%s:%s/%s", host, port, JobsPath)
```

Comment:
Note: this hardcoded protocol might prove a problem in the future.

Reply:
lets use a single parameter `--server-url http://example.org:834`

Reply:
That'd make sense, but let's do it some other time. Want to open an issue to track this?